### ISSUE TITLE
Class, record, and struct accessibility & Class, record, and struct member accessibility – improper differentiation 

### ISSUE THREAD

**Issue by bravequickcleverfibreyarn** [dotnet/docs#27155]
Since C# there can be distinguished between:
• class
• struct
• record (class)
• record struct

Also there are no extra accessibility rules that would make difference between record and corresponding _classic_ type that is based upon so there is no reason to make difference or there should be exact enumeration. Like in table or similar.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8121f526-cfe4-091d-d031-2cdc274cd8ee
* Version Independent ID: 76a389f6-b568-3568-6483-6df440f96f0c
* Content: [Access Modifiers - C# Programming Guide](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers)
* Content Source: [docs/csharp/programming-guide/classes-and-structs/access-modifiers.md](https://github.com/dotnet/docs/blob/main/docs/csharp/programming-guide/classes-and-structs/access-modifiers.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

---
[Associated WorkItem - 227655](https://dev.azure.com/msft-skilling/Content/_workitems/edit/227655)

**Comment by BillWagner** [comment]
@Uzivatel919 

I think what you're driving at is that a `record`, or `record class` *is a* `class`, and a `record struct` *is a* `struct`. Therefore the accessibility rules for `class` and `struct` apply. Yes, that can be clarified.

If you mean something different, let me know,.

**Comment by Uzivatel919** [comment]
@BillWagner, no, no you’re right.
